scraper_service/main.py

```diff
@@ -1,79 +1,90 @@
+import json
+import logging
 from datetime import datetime
 from time import sleep
+
+from pytz import timezone
```

Comment:
Let's not use this for now, it's not native to AWS and therefore hard to import

Reply:
All set